samples/UI/GameMenu/Assets/Shared/SplashScene.xkscene

```diff
@@ -19,14 +19,18 @@ Hierarchy:
                         Rotation: {X: 0.0, Y: 0.0, Z: 0.0, W: 1.0}
                         Scale: {X: 1.0, Y: 1.0, Z: 1.0}
                         Children: {}
-                    ac2b7de7553bbc4ba5bbd9c3ba319cc1: !GameMenu.SplashScript,GameMenu.Game
-                        Id: e77d2bac-3b55-4bbc-a5bb-d9c3ba319cc1
                     f6b6b1a8d48a8045bd940370455f3450: !UIComponent
                         Id: a8b1b6f6-8ad4-4580-bd94-0370455f3450
                         Page: 62c493c2-13ea-4fac-bbaf-578afafac104:SplashUI
                         Resolution: {X: 640.0, Y: 1136.0, Z: 1000.0}
                         Size: {X: 1.0, Y: 1.0, Z: 1.0}
                         ResolutionStretch: FixedWidthFixedHeight
+                    fd54344b6ac8b2ed232d62dc08820239: !GameMenu.SplashScript,GameMenu.Game
+                        Id: c7c92d4b-2c04-446f-b8c5-59e9d2b989cd
+                        NextSceneUrl: 9294c606-6173-4215-a8dc-0d5043d68333:MainScene
+                    2f704fab49b226562874d765af6b6918: !GameMenu.SplashScript,GameMenu.Game
```

Comment:
I suppose it's been added twice as mistake?

Reply:
Hmm, not sure about that. Hope I haven't introduced some bug.

Reply:
I do kind of remember having some issues while updating the samples. Game Studio wasn't playing nice when I was changing code. Maybe I tried removing/adding the component while trying to do it. I would just remove it but you all ready merged the change.